tests/admin_views/tests.py

```diff
@@ -918,6 +920,15 @@ def test_adminsite_display_site_url(self):
         self.assertEqual(response.context['site_url'], '/my-site-url/')
         self.assertContains(response, '<a href="/my-site-url/">View site</a>')
 
+    @override_settings(TIME_ZONE='America/Sao_Paulo', USE_TZ=True)
+    def test_date_hierarchy_timezone_dst(self):
+        date = pytz.timezone('America/Sao_Paulo').localize(datetime.datetime(2016, 10, 16, 15), is_dst=None)
+        q = Question.objects.create(question="Why?", posted=date)
+        Answer.objects.create(question=q, answer="Because.")
+        url = reverse('admin:admin_views_answer_changelist')
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
```

Comment:
How about using `assertContains()` to also check the date hierarchy value in the template?